server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -393,33 +398,29 @@ public short getDefaultReplication(Path path) {
   protected static Configuration getVolumeManagerConfiguration(AccumuloConfiguration conf,
       final Configuration hadoopConf, final String filesystemURI) {
 
-    Map<String,String> volumeHdfsConfigOverrides =
-        conf.getAllPropertiesWithPrefixStripped(Property.INSTANCE_VOLUME_CONFIG_PREFIX).entrySet()
-            .stream().filter(e -> e.getKey().startsWith(filesystemURI + ".")).collect(
-                Collectors.toUnmodifiableMap(e -> e.getKey().substring(filesystemURI.length() + 1),
-                    Entry::getValue));
+    final var cacheKey = new Pair<>(hadoopConf, filesystemURI);
+    return HDFS_CONFIGS_FOR_VOLUME.get(cacheKey, (key) -> {
 
-    if (volumeHdfsConfigOverrides.isEmpty()) {
-      // There are no overrides in the AccumuloConfiguration for this volume, return the Hadoop
-      // Configuration input parameter.
-      return hadoopConf;
-    } else {
-      // Get the Hadoop Configuration object from the Cache for this filesystemURI. If no object
-      // exists, then create one, apply the overrides and put it in the cache. The Volume config
-      // overrides cannot change, so we only need to cache this once. Construct a Key that is
-      // unique to the Hadoop Configuration input parameter in case different objects are passed in.
-      final String key = hadoopConf.hashCode() + filesystemURI;
-      return HDFS_CONFIGS_FOR_VOLUME.get(key, (fs) -> {
-        log.debug("Caching a new configuration for volume: {} with overrides: {}", filesystemURI,
-            volumeHdfsConfigOverrides);
-        Configuration volumeConfig = new Configuration(hadoopConf);
-        volumeHdfsConfigOverrides.forEach((k, v) -> {
-          log.info("Overriding property {}={} for volume {}", k, v, filesystemURI);
-          volumeConfig.set(k, v);
-        });
-        return new Configuration(volumeConfig);
+      Map<String,String> volumeHdfsConfigOverrides =
+          conf.getAllPropertiesWithPrefixStripped(Property.INSTANCE_VOLUME_CONFIG_PREFIX).entrySet()
+              .stream().filter(e -> e.getKey().startsWith(filesystemURI + "."))
+              .collect(Collectors.toUnmodifiableMap(
+                  e -> e.getKey().substring(filesystemURI.length() + 1), Entry::getValue));
+
+      // use the original if no overrides exist
+      if (volumeHdfsConfigOverrides.isEmpty()) {
+        return hadoopConf;
+      }
+
+      log.debug("Caching a new configuration for volume: {} with overrides: {}", filesystemURI,
```

Comment:
This logging seems to duplicate the information that will be logged in the later log.info at the debug level.  Seems like it could be removed